.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Before submitting
+
+Before submitting a PR, please make sure to check our contributing guidelines regarding [code formatting](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#code-formatting-and-typing), [tests](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#unit-tests), and [documentation](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#documentation).
+
+## What does this PR do?
+
+Fixes # (issue).
```

Comment:
There isn't always an issue to fix / close, so I would remove that.

Reply:
so just add a link to the GH keywords?
```suggestion
Closes # (issue).
```